### PR TITLE
Added Timezone selection

### PR DESCRIPTION
**Pull request by follower46**

### CODE REVIEW

File: Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py [hunk 1]
```diff
@@ -0,0 +1,334 @@
+# -*- coding: utf-8 -*-
```
Comment (follower46) [inline Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py]: I obviously stole a lot of your code

File: Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+                self._MyStack.Pop()
+                self.SyncList( self._MyStack.Last() )
+                self._PsIndex = 0
+            else:
+                self._MyStack.Push( self._MyList[self._PsIndex]._Path )
+                self.SyncList( self._MyStack.Last() )
+                self._PsIndex = 0
+                
+        if cur_li._MyType == ICON_TYPES["FILE"]: ## set the current timezone
+            subprocess.call(['sudo', 'cp', cur_li._Path, '/etc/localtime'])
```
Comment (follower46) [inline Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py]: Here is where my primary concern is, security wise.
If you know a better way to do this, I would appreciate some advice.

File: Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+        if event.key == CurKeys["Up"]:
+            self.ScrollUp()
+            self._Screen.Draw()
+            self._Screen.SwapAndShow()
+        if event.key == CurKeys["Down"]:
+            self.ScrollDown()
+            self._Screen.Draw()
+            self._Screen.SwapAndShow()
+        
+        if event.key == CurKeys["Right"]:
```
Comment (follower46) [inline Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py]: I really liked this idea.
It's especially handy on these long lists.

File: Menu/GameShell/10_Settings/Time/pages.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*- 
+
+from timezone_lib_list_page import TimezoneListPage
+import myvars
+
+def InitTimePage(main_screen):
+    myvars.TimePage = None
```
Comment (follower46) [inline Menu/GameShell/10_Settings/Time/pages.py]: I would like to add a generic clock page with the options for 12 or 24 hour display.
I haven't yet looked into the code for the top-bar, though.

File: Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+
+
+        self._Scroller = ListScroller()
+        self._Scroller._Parent = self
+        self._Scroller._PosX = self._Width - 10
+        self._Scroller._PosY = 2
+        self._Scroller.Init()
+        
+
+    def ScrollUp(self,Step=1):
```
Comment (follower46) [inline Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py]: What are  your thoughts on having a common "scroll list" class in the sys.py/UI?
I can see this being handy and I wouldn't want to copy/paste this logic every time for a list view.

Comment (cuu) [reply Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py]: why is your Menu/GameShell/10_Settings/Time/list_item.py  a binary file?

Comment (cuu) [reply Menu/GameShell/10_Settings/Time/timezone_lib_list_page.py]: do you mean sys.py/UI/scroller.py?
It's a common UI element used by

>  ./Menu/GameShell/Music Player/play_list_page.py
> ./Menu/GameShell/Music Player/music_lib_list_page.py
> ./Menu/GameShell/10_Settings/ButtonsLayout/__init__.py
> ./Menu/GameShell/10_Settings/Airplane/__init__.py
> ./sys.py/UI/Emulator/rom_list_page.py

for scroll content
maybe Dont need to copy and pase everytime ,
just 
import it

File: Menu/GameShell/10_Settings/Time/myvars.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+
+
```
Comment (Morgan-jarry) [inline Menu/GameShell/10_Settings/Time/myvars.py]: These newlines are not necessary.

Comment (follower46) [reply Menu/GameShell/10_Settings/Time/myvars.py]: Yup, agreed. I just wanted to maintain the current style from the other files.

**Pull request merged**